MS5611.cpp

```diff
@@ -129,17 +130,17 @@ void MS5611::reset()
 }
 
 
-int MS5611::read(uint8_t bits)
+int MS5611::read()
 {
```

Comment:
```
int MS5611::read(uint8_t bits)
int MS5611::read()
```
this breaks the interface so this needs some thoughts.

Your goal is to set the oversampling rate once and just call read() without parameters.
(as far as I understand)